docs/Performance.md

```diff
@@ -0,0 +1,304 @@
+# Protobuf Perforamcne
```

Comment:
Change the name of this file to doc/performance.md (lower-case p).

Reply:
Done